### PR TITLE
fix: local variable 'params' referenced before assignment

### PR DESCRIPTION
**Pull request by a13zen**
*Issue #, if available:*
Fix indent issue causing variable referenced before assignment errors

### CODE REVIEW

File: seedfarmer/commands/_deployment_commands.py [hunk 1]
```diff
@@ -499,20 +499,20 @@ def _exec_destroy(args: Dict[str, Any]) -> Optional[ModuleDeploymentResponse]:
...
-                    for dep_resp_object in destroy_response:
-                        if dep_resp_object and dep_resp_object.status in ["ERROR", "error", "Error"]:
-                            _logger.error("At least one module failed to destroy...exiting deployment")
-                            print_errored_modules_build_info(
-                                "The following modules had errors destroying ", destroy_response
-                            )
-                            raise seedfarmer.errors.ModuleDeploymentError(
-                                error_message="At least one module failed to destroy...exiting deployment"
-                            )
+                            destroy_response = list(workers.map(_exec_destroy, params))
```
Comment (dgraeber) [inline seedfarmer/commands/_deployment_commands.py:502]: This is not correct....the loop is not iterating over each module

**Review by dgraeber — CHANGES_REQUESTED** [review]

File: seedfarmer/commands/_deployment_commands.py (hunk 1 above)
Comment (dgraeber) [inline seedfarmer/commands/_deployment_commands.py:502]: This is not correct...we need to iterate at the group level...not the module level...

**Review by dgraeber — CHANGES_REQUESTED** [review]
Lets discuss the issue you are having

**Comment by a13zen** [comment]
Closing in favour of safer alternative fix to be introduced by @dgraeber

File: seedfarmer/commands/_deployment_commands.py [hunk 2]
```diff
@@ -499,20 +499,20 @@ def _exec_destroy(args: Dict[str, Any]) -> Optional[ModuleDeploymentResponse]:
...
+                            _logger.debug(destroy_response)
+                            print_modules_build_info("Build Info Debug Data", destroy_response) if _logger.isEnabledFor(
```
Comment (dgraeber) [inline seedfarmer/commands/_deployment_commands.py:504]: I have tested this..it fails due to the ordering of the deletion of module roles

**Pull request closed without merging**